src/clean.py

```diff
@@ -17,7 +17,7 @@ def to_snake(str_in, scream=False):
 if __name__ == "__main__":
 
     # read in data
-    df = pd.read_csv("resources/ames_housing.csv")
+    df = pd.read_csv("resources/data/ames_housing.csv")
 
     # convert column names to snake_case
```

Comment:
Perhaps it would be nice to centralise all of the path in a yaml or json and get them form there.